scheduler/src/cook/mesos/api.clj

```diff
@@ -668,55 +668,59 @@
     group))
 
 
-(defn get-executor-states-impl
-  "Builds an indexed version of all executor states on the specified slave. Has no cache; takes
+(defn get-executor-id->directory-impl
+  "Builds an indexed version of all executor-id to directory on the specified agent. Has no cache; takes
```

Comment:
Renamed to `sandbox-directory`. It is the path to the working directory of the task/executor, aka the sandbox directory, available from the `MESOS_DIRECTORY` environment variable on the agent.